src/Sentry.Bindings.Cocoa/Sentry.Bindings.Cocoa.csproj

```diff
@@ -54,12 +54,13 @@
   </Target>
 
   <!-- Generate bindings -->
-  <Target Name="_GenerateSentryCocoaBindings" AfterTargets="_BuildSentryCocoaSDK" Condition="$([MSBuild]::IsOSPlatform('OSX'))"
-    Inputs="..\..\modules\sentry-cocoa\Carthage\.built-from-sha" Outputs="ApiDefinitions.cs;StructsAndEnums.cs">
+  <Target Name="_GenerateSentryCocoaBindings" AfterTargets="_BuildSentryCocoaSDK" Condition="$([MSBuild]::IsOSPlatform('OSX'))">
```

Comment:
Oh, just noticed the input/output got removed. This means it'll run on each build, even if no changes were done. Was that the intention? @jamescrosswell ?

Was the `git submodule update` approach slower?

Reply:
It was intentional yes. It will make the runs somewhere between 0.5s and 2s slower, which I figured wasn't consequential. We could potentially run `git submodule update --recursive` before all of this... I'm not sure if we'd continue to run into (other) problems though and it seems like a lot of complexity to add just to shave ~1s off the build time though. I'd rather remove the optimsation and be done with it.

Reply:
Sounds good then. Surprised it takes only 2 seconds to run obj sharpie.

Reply:
Yeah it's taking about 2.5s of user time on my machine today (although only 0.67s of system time):
![image](https://github.com/getsentry/sentry-dotnet/assets/728212/e45c12d9-0d24-49e0-83c6-04b614b68a02)